examples/c_api/tiledb_examples.h

```diff
@@ -0,0 +1,57 @@
+#ifndef TILEDB_EXAMPLES_H
+#define TILEDB_EXAMPLES_H
+
+#include <stdio.h>
+#include <tiledb/tiledb.h>
+
+/**
+ * Attempt to retrieve an error from the tiledb context
+ * and print to stderr if present.
+ *
+ * @param line the line number of the last API call
+ * @param ctx the context pointer
+ *
+ * @return TILEDB_OK if no error was found, TILEDB_ERR if one was.
```

Comment:
This can return raw 0 or 1, it's not necessary to use the TileDB status codes.

Reply:
d110916